openhands/utils/conversation_utils.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
```

Comment:
this utils folder is getting out of hand 😓 probably another thing to refactor in the future